helpers/hasCorrectNextConfig.js

```diff
@@ -12,9 +11,29 @@ const hasCorrectNextConfig = async ({ nextConfigPath, failBuild }) => {
   const isValidTarget = acceptableTargets.includes(target)
   if (!isValidTarget) {
     console.log(
-      `Your next.config.js must set the "target" property to one of: ${acceptableTargets.join(', ')}. Update the
-      target property to allow this plugin to run.`,
+      `The "target" config property must be one of "${acceptableTargets.join(
+        '", "',
+      )}". Building with "serverless" target.`,
     )
+
+    /* eslint-disable fp/no-delete, node/no-unpublished-require */
+
+    // We emulate Vercel so that we can set target to serverless if needed
+    process.env.NOW_BUILDER = true
+    // If no valid target is set, we use an internal Next env var to force it
+    process.env.NEXT_PRIVATE_TARGET = 'serverless'
```

Comment:
I believe some automated tests might be using the same process, i.e. running one test would change `process.env` for other tests. We could probably fix this by either:
  - Spawning those tests in different processes
  - Loading `next` right away so it uses those environment variables, then deleting `process.env.NOW_BUILDER` and `process.env.NEXT_PRIVATE_TARGET` (or restoring them to their original value).